src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -894,15 +919,63 @@ only screen and (min-device-pixel-ratio: 2) {
 
 .tribe_welcome_page p,
 .tribe_update_page p {
+	font-size: 15px;
+}
+
+.tribe_welcome_page li {
 	font-size: 14px;
+	margin-bottom: 10px;
 }
 
 p.tribe-welcome-message {
-	font-size: 20px;
+	font-size: 18px;
 	font-weight: 400;
 }
 
-.tribe_welcome_page h2:before,
+.tribe_welcome_page h2 {
+
+	/* "Calendar" dashicon */
+	&:before {
+		color: #555d66;
+		content: '\f145';
+		font-family: 'dashicons';
+		font-size: 21px;
+		line-height: 1;
+		margin-right: 10px;
+		position: relative;
+		top: 2px;
+	}
+
+	/* "Life preserver" dashicon */
+	&[data-tribe-icon="dashicons-sos"] {
+		&:before {
+			content: '\f468';
```

Comment:
Not a thing that needs fixing now, but these should really all be set in variables somewhere so they can be referenced by a recognizable name.